client-go/LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright 2014 Engine Yard, Inc.
```

Comment:
Thanks. Just have been a part of projects in the past that are adamant about keeping that copyright year up to date.